kolibri/plugins/coach/assets/src/vue/groups-page/delete-group-modal.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+
+  <core-modal :title="$tr('deleteLearnerGroup')"
+    @cancel="close">
+    <p>{{ $tr('areYouSure') }} <strong>{{ groupName}}</strong>?</p>
```

Comment:
Hrm, this is my bad. We can translate strings that include HTML, I just haven't documented it well.

Also, because you are infilling a variable, you would need to make a method that calls `this.$trHtml('areYouSure', {groupName: this.groupName})`.

Where the `areYouSure` string would be: `'<original text> <strong> { groupName } </strong>?'`